AirCasting/SDSync/MobileSessions/SDCardMobileSessionsSavingService.swift

```diff
@@ -207,6 +210,47 @@ class SDCardMobileSessionsSavingService: SDCardMobileSessionssSaver {
         }
     }
     
+    private func addMeasurementLocation(deviceID: String, measurements: SDCardMeasurementsRow) -> CLLocationCoordinate2D? {
+
+        let isMini = deviceID.lowercased().contains(AirBeamDeviceType.airBeamMini.rawName)
+        
+        if isMini {
+            guard let sessionUUID else {
```

Comment:
Does this object "SessionSavingService" even make sense without sessionUUID in place? If not, can we easily get rid of this optional?

Reply:
It might not be that easy, we do get the sessionUUID from the file when we process it. 
We could pass it as an argument instead of using the guard statement. Is that what you have in mind?

Reply:
For future: we agreed it requires not-so-small refactor so it's added as a separate task in the trello